api/sql/schema/350-rule.conditionProperty.sql

```diff
@@ -3,8 +3,9 @@ CREATE TABLE [rule].[conditionProperty] (
     factor CHAR(2) NOT NULL,
     [name] NVARCHAR(50) NOT NULL,
     [value] NVARCHAR(200) NOT NULL,
-    CONSTRAINT pkRuleConditionProperty PRIMARY KEY CLUSTERED (conditionId, factor, [name]),
+    CONSTRAINT pkRuleConditionProperty PRIMARY KEY CLUSTERED (conditionId, factor, [name], [value]),
```

Comment:
There was an error in the primary key, which prevented repeating the tag name with different values.